### ISSUE TITLE
DEFAULT_ENDPOINT and TIMETABLE_URI in bus_stop_chooser and stop_timetable should be config parameters

### ISSUE THREAD

**Issue by abrahammartin** [ijl20/tfc_web#145]
These are the API endpoints (currently pointing at https://smartcambridge.org/transport/api). These should really be parameters that gets passed when initiating the widgets or map choosers, so we can pass different endpoints. Otherwise we can't use other API endpoints for dev and test.

**Comment by jw35** [comment]
In the bus stop chooser, DEFAULT_ENDPOINT can already be overrriden by api_endpoint from the configuration parameters for exactly this reason.

```
var api_endpoint = params.api_endpoint || DEFAULT_ENDPOINT;
```

Further, widget_config.js which implements the various chooser used (da1fcfa) to have code to set `params.api_endpoint` via the Django settings variable passed in to it's params array but that code has been removed from the current version.

**Comment by jw35** [comment]
I think this is effectively a duplicate of #122. 

The curent widget interface spec includes the following as a field in the config object to support all of this:

> settings: an object containing a copy of configuration parameters inherited from the enclosing framework's setup. Only parameters with names starting SMARTPANEL_ are included [object, required]. Currently use values: * SMARTPANEL_TRANSPORT_API: URL base for the TFC transport API * SMARTPANEL_RT_API: URL base for the real time bus traffic data

I've no idea if it's implemented, and I assume it's not being used.

**Comment by jw35** [comment]
Closed by #223
